src/flypad-backend/src/Aircraft/AircraftPreset.cpp

```diff
@@ -38,7 +38,7 @@ void AircraftPreset::onUpdate(double deltaTime) {
     // needs to be initialized
     if (!loadingIsActive) {
       // check if procedure ID exists
-      vector<ProcedureStep*>* requestedProcedure = procedures.getProcedure(loadAircraftPresetRequest);
+      const auto* requestedProcedure = procedures.getProcedure(loadAircraftPresetRequest);
```

Comment:
This variable is later assigned to the member `currentProcedure` which would remove constness so you also need to change the type of `currentProcedure`.

Reply:
That is already the case otherwise it wouldn't compile. See [your comment here](https://github.com/flybywiresim/a32nx/pull/7574#discussion_r1008314075)

Reply:
Yes you're right.